data/gegno/gegno.txt

```diff
@@ -180,3 +180,47 @@ phrase "gegno united"
 		"Dueyoh"
 		"Uuftei"
 		"Ootyff"
+
+
+# Untranslated Hails.
+
+phrase "gegno untranslated"
+	word
+		"Dudyei deiar tii cie dueyu."
+		"Udiyh tei tdiser, yye dudyei."
+		"Yye dudyei tei piyepuo."
+		"Tii duieml emm taei ooutei."
+		"Yye dudyei udiei tii ayyr dueyu."
+		"Tii keiou uf Oukeug."
+		"Oukeug keiou uf teim."
+		"Tii ayyr eno Gegno."
+		"Vytii ayyr eisktici uf tii."
+		"Scienn ayyr eisktici uf tii."
+
+
+# Mining job dialog.
+
+phrase "gegno mining payment dialog"
+	word
+		"Following a small debate,"
+		"After you depart from your ship,"
+		"Almost absent-mindedly,"
+		"Without paying you the slightest attention,"
+		
+	word
+		" a few bulky Gegno "
+		" a group of irritated Gegno "
+		" some unenthused Gegno workers "
+		" a large group of Gegno "
```

Comment:
Hulking implies that they are Iaculex (most of the Vi). Most workers at this station would be Satequli (mostly civilians) so I want to avoid more than a few size comparisons (already have the "bulky" option above, this one is for a larger group of workers, not a couple).